BP/common/culture/cultures/00_chinese.txt

```diff
@@ -0,0 +1,577 @@
+		}
+		female_names = {
+			Anzong Wenshou Aiqin Que Daoan Maoying Qigui Rongji Huinan Yuying Yingwu Xianyuan Lingyuan Wanying Yingmei Zhenfeng Miaodeng Farong Yuzhu Jiangui Fanjing Daozheng Zhirong Huizhao Shuyi Shuyuan Jieyu Ronghua Zhaoyi Lanying Baoming Jingying Shaoming Huiduan Lingqu Yu_er Nizi Jinghui Shunhua Shangrou Hui Lingguang Lingying Lingbing Zhaopei Yao_er Miaorong Shaoji Jingyan Lihua Wuhua Xianji Shouji Zhaorong Ying Xianzi Puxian Lian Mingxiang Yuanhua Tong_e Mingyue Zhaojun "Zu_e" Yuguang Zuqi Zhonghua Yuyi Changyi Huanghua Xiaolian "E_zi" Manyue Yueyi Yueshang Qieluo Lingji Zhou Shuzhen Gui Hui Huitong "Wan_er" Shun Fangmei Zhenyi Caiping Yuhuan Qiniang Qiu Qiuniang Jianrong Chongyan Zhenniang Yuniang Fujin Shiguang "E_huang" "NU_ying" Xinyue Hanyue Feishan Shunxuan Taihua Dexiu Jinfeng Chunyan Qiongxian E Taotao Bingyi Yunu Zuimei Chunluo Fengniang Daoqing Zhuozhen Yan Yanyan "Shi_er" "Xian_e" Nanxian Shanxiang Ai Shanli Jinluan "Xi_er" "Zhen_er" Wanyi Mantang Xijie Baozhu Xiuying Zhongyang Ruoxian Ruozhao Ruoxin Ruoxun Ruolun Miaozhen Mao_er Daoyi Niangzi Saihua Guiying Shengqiong Chaochao Fu Zhaoyun Rouniang Rounu Yuniang Biniang Rui Jinhua Guilan Shanwei Suyue Yunxian Nenniang Xiang Zhenli Cuiqiao Zhihui Lingxuan Jicong Shuzhu Yujing Yuanyuan Xiaowan Hengbo Mei Xiangjun Rushi Xianglan Ruying Yuhua Yuantong Yanxiang Xiurong Shouying Fuyuan Luzhen Ruirong Suzhen "Xiao_e" Xiaoying Xiaoyuan Xiaoji Xiuui "NU_jie" Qingyu Zhichong "Yi_an" Youwu Yupan Jinnu Jinluo Fujin "Hu_er" Jinxian Qiaoyun Sanjin Xiangyun Zhuzhu "Jin_er" Chuanzhu Jinzhu Xiaojin Huimei Miaoyu Quyi Siniang Sanniang "A_wu" Lizhi Jing Shu Lingyue Xianhui Jijiang "Guo_er" Huawan Huazhuang Chang Dehua Panpan Caichun Wenxiu Hong Honghong "A_man" Huimei Shuozhen Youwei Huilan Lize Mingda Sujie Yiyang Hua Yuhuan Shi Diaochan Zheng Liu Kangyi Feiyan Hou Jiangnu Na Shengtong Zhi
+		}
+		
+		dynasty_title_names = no
+
+		ethnicities = {
+			10 = asian
+		}
+	}
+
+	# WTWSMS
+
+	chinese = {
```

Comment:
I think Han is actually better here, isn't this one redundant? We integrated it from VIET before vanilla had added Han.

Reply:
Yeah